cert/kube_certs.go

```diff
@@ -153,7 +153,7 @@ func certList(CertPath, CertEtcdPath string) []Config {
 	}
 }
 
-type SeadentCertMetaData struct {
+type sealerCertMetaData struct {
```

Comment:
using sealer to name a struct is not pretty good, using CertMetaData(may be lint failed) or MetaData instead